tests/wasm.rs

```diff
@@ -359,6 +360,8 @@ async fn test_get_put_del() {
     assert_eq!(dispatch::<_, String>(db, "close", "").await.unwrap(), "");
 }
 
+/*
```

Comment:
Probably better to `#[ignore]` it that way we the code doesn't rot and we get a reminder that it is ignored.